static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { mocked } from 'ts-jest/utils';
+import raven_ from 'lib/raven';
```

Comment:
I don’t think you need to import raven or config, if you’re not using them explicitly and you have mocks for them.

Reply:
This is using the existing pattern elsewhere in the codebase for `mockReturnValue`.
We do need an explicit handle on the module later to enforce return values with `mockReturnValue`.
However I suspect you're right in that we might be able to use jest's auto mocking.
I'll have a look at simplifying and if possible raise another PR.

Reply:
Et voila: https://github.com/guardian/frontend/pull/23887